documentation/internal/Filters-for-Applications.md

```diff
@@ -57,59 +62,56 @@ The `user-role` is used to determine how the columns and rows should be displaye
 #### type
 
 **This is a compulsory filter**
-In case none is included in URL will be deduced from 1st user's persmissions
+In case no `type` is included in URL, the type will be deduced from 1st user's persmissions.
 
-Example: `type=user-registration`  
-Considered in implementation: **Yes** [#131 - Headers]  
+Included: **Yes**
 Can have combined values: **No**
-
-Options:
-
-- `template-code` - Of application type in list (and header)
+Examples: `type=user-registration`
+After is converted to Graphql: `templateCode: { equalToInsensitive: user-registration}`
 
 <a name="category"></a>
 
 #### category (_not yet implemented in schema_)
 
-Example: `category=company`  
-Considered in implementation: **Yes** **[New issue?]**  
+Included: **No** [Epic#82](https://github.com/openmsupply/application-manager-web-app/issues/540)  
 Can have combined values: **No**  
-Options:
-
-- `category-code` - Group of templates (application types)
+Examples: `category=company`  
+After is converted to Graphql: **TODO**
 
 <a name="stage"></a>
 
 #### stage
 
-Example: `stage=screening`  
-Considered in implementation: **Yes** [#251 - Filters]  
-Can have combined values: **Yes** e.g: `stage=screening,assessment,final-decision`  
-Options:
-
-- Set of `stage-name` which is deduced per template (application type)
+Included: **Yes**
+Can have combined values: **Yes**  
+Examples: `stage=screening` or `stage=screening,assessment`
+After is converted to Graphql: `stage: { inInsensitive: ["screening", "assessment"]}`
 
 <a name="status"></a>
 
 #### status
 
-Example: `status=draft`
-
-Considered in implementation: **Yes** [#251 - Filters]
-
-Can have combined values: **Yes** e.g: `status=draft,submitted`
+Included: **Yes**
+Can have combined values: **Yes**
+Examples: `status=draft` or `status=submitted,changes%20required`
+After is converted to Graphql: `status: {in: ["SUBMITTED", "CHANGES_REQUIRED"]}`
+Options: Set of static statuses from ApplicationStatus ENUM
 
-Options:
-
-- Set of `status` options
+- `draft` or `DRAFT`
+- `withdrawn` or `WITHDRAWN` _not implemented_
+- `submitted` or `SUBMITTED`
+- `changes required` or `CHANGES_REQUIRED`
+- `re submitted` or `RE_SUBMITTED` _not implemented_
+- `completed`or `COMPLETED`
+- `expired` or `EXPIRED` _not implemented_
```

Comment:
Update these to new values (Withdrawn and Expired moved to Outcomes)

Reply:
Good! Updating now